yarn.lock

```diff
@@ -9208,6 +9224,11 @@ crypto-js@^3.1.4:
   resolved "https://registry.yarnpkg.com/crypto-js/-/crypto-js-3.1.8.tgz#715f070bf6014f2ae992a98b3929258b713f08d5"
   integrity sha1-cV8HC/YBTyrpkqmLOSkli3E/CNU=
 
+crypto-js@^4.1.1:
+  version "4.1.1"
+  resolved "https://registry.yarnpkg.com/crypto-js/-/crypto-js-4.1.1.tgz#9e485bcf03521041bd85844786b83fb7619736cf"
+  integrity sha512-o2JlM7ydqd3Qk9CA0L4NL6mTzU2sdx96a+oOfPu8Mkl/PK51vSyoi8/rQ8NknZtk44vq15lmhAj9CIAGwgeWKw==
+
```

Comment:
Can you describe what crypto-js is used for in your package? Links to code is fine.

Reply:
I can ask our dev team for specifics (might not get a clear answer) - I don't know what crypto.js does. Here's a link to their repo. https://github.com/brix/crypto-js

Reply:
i meant how its being used in these packages, not what the library does, just to clarify

Reply:
We're using the crypto.js package for hashing algorithms: eip137 (sha3) and zns  (sha256) in the resolution library.

Reply:
https://github.com/unstoppabledomains/resolution/blob/09079329a3545d0f344585d898f9f19cac4eeb9b/src/utils/namehash.ts